### DISCUSSION TITLE
Can Homebrew formulae be configured to link against macOS's built-in LibreSSL?

### DISCUSSION THREAD

**Discussion by bgshacklett** [Homebrew/discussions#83] [Everyday usage] [answered]
I often work in scenarios where custom trusted CA certificates may need to be configured. In some cases, these may be configured via MDM, or other means, by IT departments and they already exist in the macOS keychain.

It's my understanding that the built-in LibreSSL libraries that come with macOS are able to use the keychain as a certificate store. With that being the case, it would be useful to be able to install packages via Homebrew which can use this library so that I don't have to configure my own certificates on top of what's already been provided. Normally I see references to OpenSSL in the compilation process. Can this be overridden to point to the built-in LibreSSL?

**Comment by jonchang** [comment, marked as answer]
Most of the formulae built by Homebrew links against our brewed copy of openssl, as you correctly note. This is because Apple doesn't really officially support its internal copy of libressl for use by non-Apple parties. I believe apps that want to use the system certificate store need to use Secure Transport. You could probably script something that synchronizes the system certificates with certificate files that brewed openssl understands.
